galaxy.yml

```diff
@@ -2,7 +2,7 @@
 
 namespace: lowlydba
 name: sqlserver
-version: 0.1.1
+version: 0.2.0
```

Comment:
This is fine here though I'd usually make it in a separate commit (I would typically push this directly right after a release instead of making a PR).

Reply:
Yeah I realized I wasn't sure where best to do this - I was worried about putting the `version_added` and then later using a different version than `2.0` next 🤔